scripts/build/travis/configure_travis_bionic.sh

```diff
@@ -31,6 +31,8 @@ add_app ${KRATOS_APP_DIR}/DEMApplication;
 add_app ${KRATOS_APP_DIR}/CSharpWrapperApplication;
 add_app ${KRATOS_APP_DIR}/MetisApplication;
 add_app ${KRATOS_APP_DIR}/TrilinosApplication;
+add_app ${KRATOS_APP_DIR}/PfemFluidDynamicsApplication;
```

Comment:
this one is supposed to be there?

Reply:
Yes, the Swimming DEM app solves the coupling of DEM with Fluid, but also with PfemFluid